rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/stop_scan/action.py

```diff
@@ -0,0 +1,28 @@
+import insightconnect_plugin_runtime
+from .schema import StopScanInput, StopScanOutput, Input, Output
+# Custom imports below
+import requests
+
+
+class StopScan(insightconnect_plugin_runtime.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="stop_scan",
+            description="Stop a currently running scan",
+            input=StopScanInput(),
+            output=StopScanOutput(),
+        )
+
+    def run(self, params={}):
+        scan_id = params.get(Input.ID)
+        url = f"scan/{scan_id}/stop"
+        try:
+            response = self.connection.ivm_cloud_api.call_api(url, "POST")
+            if response == 202:
+                return {Output.SUCCESS: True}
+            else:
+                return {Output.SUCCESS: False}
```

Comment:
Could we add non-required output variables here of say `status_code` and `message` where we could know why it wasn't successul?